### ISSUE TITLE
Support for mapped drives

### ISSUE THREAD

**Issue by apazzy** [ebkr/r2modmanPlus#82]
I've got a somewhat weird setup and have my games drive mapped on a NAS, but this means I can't use r2modman

If possible, please add a configuration option to copy mods instead of using a symlink.

**Comment by ebkr** [comment]
What format is the NAS? ExFAT is known to have issues.

**Comment by apazzy** [comment]
The NAS is running Unraid, with an XFS array and BTRFS cache. My games share usually sits on the array, but if I play a game I usually move it to the cache.

**Comment by ebkr** [comment]
From what I can tell, XFS should support symlinks, are you getting an EISDIR error?

**Comment by apazzy** [comment]
Dunno where to find any particular logs, but the error I get is

```
ERROR
Failed to produce a symlink between profile and RoR2
UNKNOWN: unknown error, symlink 'C:\Users\<user>\AppData\Roaming\r2modmanPlus-local\mods\profiles\Default\BepInEx' -> 'G:\Steam\steamapps\common\Risk of Rain 2\r2modman\BepInEx'
```

**Comment by ebkr** [comment]
I'll work on a fallback mode that'll copy the directories across instead, but I'm not sure how long it'll take.

**Comment by apazzy** [comment]
Thanks, no rush from me of course. For now I'm just copying stuff manually.

If you put in like an advanced features/beta options sub-menu I'll gladly test, even if it's a one-way copy type mechanism.

**Comment by ebkr** [comment]
I've released an update (3.0.6) to solve this issue.
It's currently only a one-way copy, which performs the same as the original one.

I may improve this in the future, but this should resolve this problem for now.
https://github.com/ebkr/r2modmanPlus/releases/tag/v3.0.6

Let me know if something doesn't work. The fastest way to contact me is on Discord (see README).
